src/app/pages/sharing/components/shares-dashboard/smb-card/smb-card.component.ts

```diff
@@ -36,27 +37,29 @@ import { selectService } from 'app/store/services/services.selectors';
 })
 export class SmbCardComponent implements OnInit {
   requiredRoles = [Role.SharingSmbWrite, Role.SharingWrite];
+  changedEnabled = new Map<number, boolean>([]);
```

Comment:
Let's imagine fixing every other table with this approach.
TBH, it looks like a workaround. We have a mechanism called `getSmbShares` to get fresh results on each change. The question is, why does it not work as expected?